api/graphql/makeModels.js

```diff
@@ -293,6 +324,8 @@ export default function makeModels (userId, isAdmin, apiClient) {
         details: p => p.details(userId),
         isAnonymousVote: p => p.get('anonymous_voting') === 'true',
         myReactions: p => userId ? p.reactionsForUser(userId).fetch() : [],
+        // clickthrough: p => p.pivot && p.pivot.get('clickthrough'), // TODO COMOD: does not seem to work
```

Comment:
I do think this should work, I can try to play with it

Reply:
sure thing